README.md

```diff
@@ -39,9 +39,15 @@ install the bundle via:
 ## Loki variant build
 At the moment, Loki variant cannot be compiled with GNU 11 compiler. On ATOS HPC, Loki variant may be built using
 ```
-./cloudsc-bundle build --with-loki --loki-frontend=fp --arch=./arch/ecmwf/hpc2020/gnu/9.3.0/
-./cloudsc-bundle build --with-loki --loki-frontend=fp --arch=./arch/ecmwf/hpc2020/intel/2021.4.0
+./cloudsc-bundle build --clean --with-loki --loki-frontend=fp --arch=./arch/ecmwf/hpc2020/gnu/9.3.0/
+./cloudsc-bundle build --clean --with-loki --loki-frontend=fp --arch=./arch/ecmwf/hpc2020/intel/2021.4.0
 ```
+Targetting GPU, one should use:
+```
+ ./cloudsc-bundle build --with-gpu --with-loki --loki-frontend=fp --arch=./arch/ecmwf/hpc2020/nvhpc/22.1 
+```
+NVHPC will not build Loki variant with --build-type=debug due to the compiler error. Possibly, look into the compiler flags is needed here.
+
```

Comment:
We'll also need updated run instructions here, in particular for Loki-SCC where we need `NV_ACC_CUDA_HEAPSIZE=9GB` or similar.